src/App.js

```diff
@@ -15,7 +15,17 @@ const room = '1234'
 class App extends Component {
```

Comment:
recommend replacing return with the following:
```
  return (
      <div>
      <Navbar></Navbar>
      <Router>
          <Route
            path='/chat'
            render={(props) => <Chat {...props} socket={socket} room={room}/>}
          />
          <Route path="/ask" component={AskQuestions}/>
      </Router>
      </div>
    );
```

This is just a recommendation for after and I'd recommend to do some refactoring once we know what we want the chat to look like but all and all it looks good!